docs/developer/how-to/make-new-ophyd-async-device.md

```diff
@@ -0,0 +1,58 @@
+# Decision Flowchart for Creating a New ophyd_async Device
+
+This document contains decision flowcharts designed to guide developers through the process of creating a new ophyd_async device in the Ophyd library. These flowcharts help in determining the appropriate class inheritance, methods to override, and testing procedures for optimal device functionality and integration into the system.
+
+## High-Level Development Flowchart
+
+This high-level flowchart guides the overall process of creating a new ophyd_async device, from testing PVs to overriding methods and determining state enums.
+
+```{mermaid}
+
+
+  flowchart TD
+    highLevelStart([Start]) --> scoutUsers[Scout current and potential users of the device, tag them in the draft PR]
+    scoutUsers --> testPVs[Test PVs to get the right values]
+    testPVs --> decideStateEnums[Decide on State Enums: extend str, Enum]
+    decideStateEnums --> chooseMethods[Choose which superclass methods to override]
+    chooseMethods --> finalizeDevice[Finalize Device Implementation]
+    finalizeDevice --> requestFeedback[Request feedback from tagged users]
+    requestFeedback --> rebaseOnMain[Rebase on main -> make sure tests pass still]
+    rebaseOnMain --> coordinateMerges[Coordinate merges with other codebase updates]
+    coordinateMerges --> markPRReady[Mark PR as ready]
+
+```
+
+## Interface Selection Flowchart
+
+This flowchart assists in selecting the appropriate interfaces based on the device's capabilities, such as file writing, reading values from process variables (PVs), and mobility within scans.
+
+```{mermaid}
+
+  flowchart TD
+    start([Start]) --> isFileWriting{Is it a File Writing Detector?}
+    isFileWriting -- Yes --> useStandardDetector[Use StandardDetector]
+    isFileWriting -- No --> producesPVValue{Does it produce a value from a PV you want to read in a scan?}
+    producesPVValue -- Yes --> isMovable{Is it something that you move in a scan?}
+    isMovable -- Yes --> useReadableMovable[Use StandardReadable + AsyncMovable + Override set method]
+    isMovable -- No --> useReadable[Use StandardReadable]
+    producesPVValue -- No --> useDevice[Use Device]
```

Comment:
Ditto to my other comment, I would prefer examples to flowcharts

Reply:
This flowchartis still in `ophyd_async`. I think it's correct that it stays in `ophyd_async` and stays as a flowchart. I don't think we need a copy of it in `dodal` but feel free to link it to the page in `ophyd_async`

Reply:
got a link